README.md

```diff
@@ -1,41 +1,122 @@
+### 3.  Setup SubnetId's for deploying resources    
+   - Before deploying your stack, you need to edit the `SubnetId` [here](./Templates1/cf-template.json#L22) for `cloudfix-linter-demo-cloudformation` stack and [all subnets in this file](https://github.com/trilogy-group/cloudfixLinter-demo-cloudformation/blob/newResources/README.md) for `CfDemoStack` stack to the id of a subnet that exists in your account and to which you would like to deploy the resources.
+
+ ### 4. Stack Deployment    
+**Note : - For demo purpose we have already deployed a stack with name `CfDemoStack` for using `all-resources-template.json` and `cloudfix-linter-demo-cloudformation` for using `cf-template.json` with given template in Q3 of TrilogyAccount. You can use it for demo purpose, if you are a Trilogy account holder.**       
+   - Deploy a stack in your AWS account using the template (No need to do this again if deployed once and that stack isn't deleted). There are two ways to do this:
     - Use AWS [console](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks)
+
     - Login to aws through CLI, then run this command
       ```
-      aws cloudformation deploy --template-file Templates1/cf-template.json --stack-name <stackName>
+      aws cloudformation deploy --template-file Templates1/all-resources-template.json --stack-name <stackName> --capabilities CAPABILITY_IAM
       ```
- 5. For viewing **mock recommendations**. Since Cloudfix won't give recommendations for your resources till it has observed behaviour for 14 days, so for a quick preview of the tool mock recommendations can be used to fake Cloudfix recommendations for your resources:
-    - Run command `python utils/gen_recco.py <stackName>`. For demo purpose we have already deployed a stack with name `cloudfix-linter-demo-cloudformation` with given template in Q3. You can use it for demo purpose.
-    - This will generate a reccos.json file which will be used to facilitate mock recommendations
+   - For testing on lesser resources please use cf-template.json instead of all-resources-template
+
+ ***Note : Stack creates IAM roles. Please ensure you have sufficient permissions to create IAM roles**
+ 
+ ### 5.  Getting Mock recommendations    
+   - For viewing **mock recommendations**. Since Cloudfix won't give recommendations for your resources till it has observed behaviour for 14 days, so for a quick preview of the tool mock recommendations can be used to fake 
+   #### To use mock recommendations.
+   In order to generate mock recommnedations and tell the linter that it needs to read reccomendations from a file rather than from CloudFix itself, on the terminal run
+   - Windows
+   ```
+   $env:CLOUDFIX_FILE=$true
+   python utils/gen_recco.py <stackName>
+   ```
+   - Linux and Devspaces
+   ```
+   export CLOUDFIX_FILE=true
+   python utils/gen_recco.py <stackName>
+   ```
+
+   #### To use cloudFix recommendations from via CLI
```

Comment:
This section seems to be under `5. Getting Mock recommendations

Reply:
It is under #5.
#5 is quite big it has both mock recommendation and using cli to get recommendatios under it

Reply:
Then #5 should be renamed to `Fetching recommendations` because it has both mock and non mock

Reply:
cool